etc/cert/ca-config.json

```diff
@@ -0,0 +1 @@
+{"signing":{"default":{"expiry":"43800h","usages":["signing","key encipherment","server auth","client auth"]}}}
```

Comment:
is it possible to have cert/* to be under test/resources/cert? since it is part of test?